clusters/build-clusters/01_cluster/sshd-bastion/ppc64le-libvirt/deployment.yaml

```diff
@@ -191,6 +191,101 @@ items:
       port: 443
       targetPort: https-1-3
     type: ClusterIP
+- apiVersion: v1
+  kind: Service
+  metadata:
+    namespace: bastion-ppc64le-libvirt
+    name: sshd-1-0
```

Comment:
Looking at the changes, you might wanna change the sshd-1-0 -> sshd-2-0 and similar for all of them.

Reply:
Good catch!